code/game/objects/items/rogueitems/keyrings.dm

```diff
@@ -182,16 +182,19 @@
 	keys = list(/obj/item/key/nightman)
 
 /obj/item/storage/keyring/hand
-	keys = list(/obj/item/key/keep_armory, /obj/item/key/steward, /obj/item/key/town_dungeon, /obj/item/key/town_barracks, /obj/item/key/keep_gatehouse, /obj/item/key/councillor_rooms, /obj/item/key/hand, /obj/item/key/walls, /obj/item/key/keep_dungeon, /obj/item/key/keep_barracks, /obj/item/key/manor, /obj/item/key/kitchen)
+	keys = list(/obj/item/key/keep_armory, /obj/item/key/steward, /obj/item/key/town_dungeon, /obj/item/key/town_barracks, /obj/item/key/keep_gatehouse, /obj/item/key/councillor_rooms, /obj/item/key/hand, /obj/item/key/walls, /obj/item/key/keep_dungeon, /obj/item/key/keep_barracks, /obj/item/key/manor, /obj/item/key/kitchen, /obj/item/key/prince_rooms, /obj/item/key/royal)
 
 /obj/item/storage/keyring/steward
 	keys = list(/obj/item/key/steward, /obj/item/key/walls, /obj/item/key/manor, /obj/item/key/kitchen)
 
 /obj/item/storage/keyring/servant
 	keys = list(/obj/item/key/manor, /obj/item/key/kitchen, /obj/item/key/keep_barracks)
 
+/obj/item/storage/keyring/butler
+	keys = list(/obj/item/key/manor, /obj/item/key/kitchen, /obj/item/key/prince_rooms, /obj/item/key/royal, /obj/item/key/keep_barracks, /obj/item/key/walls, /obj/item/key/keep_gatehouse, /obj/item/key/keep_armory, /obj/item/key/councillor_rooms, /obj/item/key/keep_dungeon,  /obj/item/key/keep_dungeon_torture, /obj/item/key/physician, /obj/item/key/tower)
+
 /obj/item/storage/keyring/archivist
-	keys = list(/obj/item/key/manor, /obj/item/key/archive)
+	keys = list(/obj/item/key/archive)
```

Comment:
I'm ambivalent on this change, honestly. We've been moving the scholar out of the keep for sometime, finalizing it makes some sense; But in the same notion, I think the scholar is expected to help teach the heirs? Atleast lorely anyway.